sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/providers/BigQueryStorageWriteApiSchemaTransformProvider.java

```diff
@@ -157,24 +160,41 @@ public static Builder builder() {
           .Builder();
     }
 
+    @SchemaFieldDescription(
+        "The bigquery table to write to. Format: [${PROJECT}:]${DATASET}.${TABLE}")
     public abstract String getTable();
 
+    @SchemaFieldDescription(
+        "Optional field that specifies whether the job is allowed to create new tables. "
+            + "The following values are supported: CREATE_IF_NEEDED (the job may create the table), CREATE_NEVER ("
+            + "the job must fail if the table does not exist already).")
```

Comment:
Can we add a comment about the default? I think this was it (based on https://cloud.google.com/bigquery/docs/reference/rest/v2/Job), but we should probably test this too. 

The default value is CREATE_IF_NEEDED.